BackendWebApp/src/test/java/uk/ac/ebi/biostd/webapp/application/rest/service/PendingSubmissionListFilterTest.java

```diff
@@ -0,0 +1,139 @@
+package uk.ac.ebi.biostd.webapp.application.rest.service;
+
+import org.junit.Test;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionListFiltersDto;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionListItemDto;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
fix import order and formating

Reply:
![image](https://user-images.githubusercontent.com/28758892/40832418-308919b4-6583-11e8-92ad-a20d39cd049c.png)

Reply:
Done.